### PR TITLE
Bug 1359585 edited exposing router metrics section

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1359585

@ramr , this is essentially a followup of #1352. Can I get an ack that no meaning was changed? I'm particularly concerned about the explanations of the steps that I added. Thanks!

### CODE REVIEW

File: install_config/install/deploy_router.adoc [hunk 1]
```diff
@@ -935,40 +935,50 @@ add routes across the namespaces.
...
-based router implementations and in the case of custom implementations
-enables the image to use for a custom metrics exporter (or publisher).
+The `--metrics-image` option overrides the defaults for HAProxy-based router
+implementations and, in the case of custom implementations, enables the image to
+use for a custom metrics exporter or publisher.
 ====
 
-
 ifdef::openshift-enterprise[]
+. Grab the HAProxy router image from the Docker registry:
```
Comment (ramr) [inline install_config/install/deploy_router.adoc]: Should read `grab the HAProxy prometheus exporter image ...`.

File: install_config/install/deploy_router.adoc [hunk 2]
```diff
@@ -977,15 +987,29 @@ $ oadm router \
 ====
 endif::[]
 ifdef::openshift-origin[]
+. Grab the HAProxy router image from the Docker registry:
```
Comment (ramr) [inline install_config/install/deploy_router.adoc]: Same here - should read `grab the HAProxy prometheus exporter image ...`.

**Comment by ramr** [comment]
Minor clarifications needed re: the image being pulled down - its the prometheus haproxy exporter image not the haproxy router image.  But other than that  LGTM

**Comment by bfallonf** [comment]
@ramr Big thanks. I've changed to suggestion.

@ahardin-rh @adellape @tnguyen-rh @tpoitras Peer review please?

**Comment by ahardin-rh** [comment]
@bfallonf LGTM! :fireworks:

**Comment by bfallonf** [comment]
@ahardin-rh thanks! I'll merge this one.

**Pull request merged**